SU2_PY/SU2/io/config.py

```diff
@@ -340,10 +340,23 @@ def read_config(filename):
             break
         
         # remove line returns
-        line = line.strip('\r\n')
+        line = line.strip('\r\n').strip()
+
+        if (len(line) == 0):
+            continue
         # make sure it has useful data
-        if (not "=" in line) or (line[0] == '%'):
+        if (line[0] == '%'):
             continue
+
```

Comment:
a small comment here maybe like "# Unify lines that end with a continuation char '\' "